R/biokinetics.R

```diff
@@ -151,32 +178,38 @@ biokinetics <- R6::R6Class(
       data.table::setcolorder(dt_out, c("t", "p", "k"))
 
       if (!has_covariates) {
-        dt_out[, p:= NULL]
+        dt_out[, p := NULL]
       }
       dt_out
     },
     prepare_stan_data = function() {
-      pid <- value <- censored <- titre_type <- obs_id <- time_since_last_exp <- NULL
+      pid <- value <- censored_lo <- censored_hi <- titre_type <- obs_id <- time_since_last_exp <- NULL
       stan_data <- list(
         N = private$data[, .N],
         N_events = private$data[, data.table::uniqueN(pid)],
         id = private$data[, private$pid_lookup[pid]],
         value = private$data[, value],
-        censored = private$data[, censored],
         titre_type = private$data[, private$titre_type_lookup[titre_type]],
         preds_sd = private$preds_sd,
         K = private$data[, data.table::uniqueN(titre_type)],
-        N_uncens = private$data[censored == 0, .N],
-        N_lo = private$data[censored == -2, .N],
```

Comment:
There was actually a bug here - I releved the censoring indicator so that -1, 0 and 1 were the levels, but didn't update this bit of the code [here](https://github.com/seroanalytics/epikinetics/commit/8632ba84064a40a85c433d16c9a5d4b7e5301ee9).